torch/_inductor/codegen/common.py

```diff
@@ -1738,7 +1738,9 @@ def inner(*args, **kwargs):
                     value = getattr(parent_handler, name)(*args, **kwargs)  # type: ignore[has-type]
 
                     def do_cse(v):
-                        csevar = self.cse.generate(self.compute, v, bounds=bounds)
+                        csevar = V.kernel.cse.generate(
+                            V.kernel.compute, v, bounds=bounds
+                        )
```

Comment:
Makes sure we are working on the current kernel when the nested kernel is being generated, e.g., CppKernel generated as the epilogue of CppTemplateKernel.